app/services/participants/query.rb

```diff
@@ -52,7 +52,7 @@ def where_from_participant_id_is(from_participant_id)
     end
 
     def order_by
-      sort_order(sort:, model: User, default: { created_at: :asc })
+      sort_order(sort:, model: User, default: Arel.sql("COALESCE(applications.accepted_at, users.created_at) ASC"))
```

Comment:
I don't think we need to use `COALESCE` here as we're only looking at accepted applications.
The field `accepted_at` should be set, so we could just order by `{ accepted_at: :asc }`